src/test/java/com/geronimo/GeronimoDatabaseTest.java

```diff
@@ -0,0 +1,41 @@
+package com.geronimo;
+
+import com.geronimo.dao.UserRepository;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import javax.persistence.EntityManager;
+import javax.persistence.EntityManagerFactory;
+import javax.persistence.Query;
+import java.math.BigInteger;
+
+import static org.junit.Assert.assertEquals;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+public class GeronimoDatabaseTest {
+
+    private UserRepository userRepository;
+    private EntityManagerFactory factory;
+
+    @Test
+    public void testDatabase() {
```

Comment:
Great test! I suggest you to choose more informative names for tests. e.g. "testDatabaseConnectionEstablished"